packages/fiori-elements-writer/templates/v2/common/extend/webapp/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "sap.ui5": {
         "models": {
-            "": {
+            "<%- defaultModel %>": {
```

Comment:
As discussed for non OVP `feApp.service.model` should be empty when written to the models key in the manifest.json

Reply:
If a model name is provided through the interface: 
```
...
    service: {
        model: string;
    }
...
```
Wouldn't a consumer expect that is the model name used in all cases? I can update the model name to be '' for the case of non-OVP templates - I wasn't sure we should be overriding the consumers settings.

Reply:
I have removed the `service.model` from the `FioriElementsApp` interface, this will clearly signal to consumers that their model name will not be used for FE apps. Thx @devinea.